### PR TITLE
feat: APP-375 update /profile routes to be under /dashboard and /dashboard/admin

### PR DESCRIPTION
**Pull request by r41ph**
## Description

https://regennetwork.atlassian.net/browse/APP-375

### Author Checklist

- [x] provided a link to the relevant issue or specification
- [x] provided instructions on how to test
- [x] reviewed "Files changed" and left comments if necessary
- [x] confirmed all CI checks have passed

### How to test

1. https://deploy-preview-2505--regen-marketplace.netlify.app/ 
2. Log in and verify that all links work as expected in the dashboard, edit profile side nav and main nav as stated in the Jira issue

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

**Comment by r41ph** [comment]
@erikalogie see testing instructions

**Comment by erikalogie** [comment]
LGTM

**Comment by blushi** [comment]
could you update this branch to dev and make necessary changes for the stuff that has been added as part of https://github.com/regen-network/regen-web/pull/2486/files?

**Review by blushi — CHANGES_REQUESTED** [review]
Looks like we're missing a few existing links to `/profile` still:
`useOnProfileClick`
`ProfileEdit.Main`
`ProfileEdit.ViewProfile`

File: web-marketplace/src/pages/Dashboard/Dashboard.constants.tsx [hunk 1]
```diff
@@ -22,45 +22,45 @@ export const SEPARATOR = {
...
+  href: '/dashboard/admin/profile',
   icon: <EditIcon className="w-[24px] h-[24px] text-[18px]" linearGradient />,
   labelClassName,
   className,
   disabledTooltipText: NOT_SUPPORTED_TOOLTIP_TEXT,
 };
 export const PROFILE_SETTINGS = {
   label: msg`Settings`,
-  href: '/profile/edit/settings',
+  href: '/dashboard/admin/settings',
```
Comment (blushi) [inline web-marketplace/src/pages/Dashboard/Dashboard.constants.tsx:63]: Also the href should be updated for `MY_ORDERS, `MY_PREFINANCE_PROJECTS`, `SAVED_PAYMENT_INFO` below

**Review by blushi — APPROVED** [review]

**Pull request merged**